hedyorg/hedy
# [UI] Improves create multiple accounts UI

**Description**
In this PR we improve the UI of the multiple accounts creation page. The following changes are made:
- When pressing `+` 5 rows are added instead of one
- The `generate passwords` button is replace by a toggle
- When passwords are generated the input field is greyed out and disabled
- Add a button to prefix all usernames with the class name

**Fixes**
This PR fixes #3774.

**How to test**
Make sure you are a teacher and have class. Verify the new functionalities by adding students by creating new accounts.
